### PR TITLE
update list with current, known instances in the wild, added 7 URLs

### PR DESCRIPTION
**Pull request by ewingson**
d2024-05-09 t20:53

doing this crosschecked with @csarven, @bourgeoa or maintainers, please review and merge

foreseeable the added providers are persistent enough

hope that helps

### CODE REVIEW

File: pages/use-solid.md
Comment (csarven) [inline pages/use-solid.md:38]: As of writing this https://web.archive.org/web/20240517095415/https://start.inrupt.com/profile :

>PodSpaces allows developers to get a WebID and a Solid Pod in order to create apps using Inrupt's Enterprise Solid Server

That clearly seems to be a provider/service intended to function as a proprietary system, effectively creating a vendor lock-in scenario. This is not aligned with the goals of the Solid Project, as exemplified by the statement above the Pod Providers table:

>With Solid, you won't be tied to the provider you choose now. That is, you have the ability to move your data elsewhere if you want.

In other words, the goal is to promote apps that can work with any Solid server conforming to the specifications developed through open standards bodies.

So, consider:

```suggestion
```

File: pages/use-solid.md
Comment (csarven) [inline pages/use-solid.md:41]: >We apologize for the inconvenience arising through this security [issue](https://github.com/nodeSolidServer/node-solid-server/issues/1768).
>Consider using [solidcommunity.net](https://solidcommunity.net/).

The current corsProxy feature is essentially missing authentication and/or moderation of some sort based on service policy. Simply don't enable corsProxy on the service.

As of this writing, solidcommunity.net offers corsProxy, so the recommendation to "consider using" it doesn't address the "issue".

File: pages/use-solid.md
Comment (csarven) [inline pages/use-solid.md:44]:
```suggestion
| [trinpod.eu](https://trinpod.eu)                   | [Graphmetrix, Inc.](https://graphmetrix.com/terms)                               | [Hetzner](https://www.hetzner.com/)           | Finland              | [TrinPod](https://graphmetrix.com/construction/trinpod)     |
```

File: pages/use-solid.md
Comment (csarven) [inline pages/use-solid.md:45]: Can't tell exactly if this is just an identity provider and/or storage, and also whether it is for individuals and/or business. But okay.

File: pages/use-solid.md
Comment (csarven) [inline pages/use-solid.md:53]: Seems like it is NSS with some OpenLink customisations? If that's a different software than NSS, it is better to link to that instead of NSS.

```suggestion
| [OpenLink Software(2)](https://solid.openlinksw.com:8445/) | [OpenLink Software](https://www.openlinksw.com/)                   | [Amazon](https://aws.amazon.com)              | USA              | OpenLink [NSS](https://github.com/solid/node-solid-server)  |
```

File: pages/use-solid.md
Comment (csarven) [inline pages/use-solid.md:52]: Seems like it is NSS with some OpenLink customisations? If that's a different software than NSS, it is better to link to that instead of NSS.

File: pages/use-solid.md
Comment (csarven) [inline pages/use-solid.md:51]: When I try to register, I'm seeing https://solid.aifb.kit.edu/noRegistrationAllowed.html

**Review by csarven — CHANGES_REQUESTED** [review]
Couple of things. Not sure about the correctness of some of this information, e.g., the software that's being used. Is that based on public information or informed by the provider? Some of the services require paid sign-ups, so can't tell.

Generally, we should aim to have the consent of the provider to be listed in this table. Or at least inform them that they are being listed so that they can have a chance to correct this information.

**Comment by ewingson** [comment]
I have edited precise.
though, not sure, how to use the github `resolve conversation` feature

File: pages/use-solid.md
Comment (melvincarvalho) [reply pages/use-solid.md:38]: > With Solid, you won't be tied to the provider you choose now. That is, you have the ability to move your data elsewhere if you want.

This is not true, and has never been true.

> This is not aligned with the goals of the Solid Project

Which goals?

> That clearly seems to be a provider/service intended to function as a proprietary system

Shouldnt solid work with both open and proprietary system?

Comment (ewingson) [reply pages/use-solid.md:38]: I have resolved everything except this one, until we have discussed / decided

**Review by ewingson — COMMENTED** [review]
how do we find consensus ?
+1 for keeping it in the list...

File: pages/use-solid.md
Comment (csarven) [reply pages/use-solid.md:38]: >This is not true, and has never been true.

If you believe this to be not true, please follow-up with a correction by making a new PR.

Moreoever, please also note that the quoted text was introduced and revised, in particular at:

* https://github.com/solid/solidproject.org/commit/c625071532507352bf4789c98fb980f3a23254a1
* https://github.com/solid/solidproject.org/commit/50fcdfccf23f84fd9e071035090f78764f3483b4

in fact by members of Inrupt. If they've made an error, please again, follow with a correction.

In this context, I was merely referring to the existing text on the website.

---

As Matthias has already considered and integrated some my other change requests, I am now dismissing my review.

Comment (michielbdejong) [reply pages/use-solid.md:38]: OK @csarven if you think this is all good now, can you give an 'approve' review and merge?

**Comment by TallTed** [comment]
@ewingson — in https://github.com/solid/solidproject.org/pull/888/commits/b98072b96c5d7ad6f7958d6f4bd28c9736c04ff4, you said --
```
- I have deleted the OpenLink servers, because the server seems no original 
  NSS and the versions seem deprecated (v4 and v5)
```

I need to understand what the above means. 
* "the server seems no original NSS" is not English. 
* "the versions seem deprecated (v4 and v5)" suggests that the list is only meant to include specific version(s) of something, but I don't see anything that says what those versions are. 

To my mind, "current, known instances in the wild" should include any version of any implementation. That should include those two OpenLink-hosted instances, which are running [OpenLink forks of NSS](https://github.com/OpenLinkSoftware/node-solid-server).

/cc @kidehen @openlink

**Comment by kidehen** [comment]
Those are instances that work, and based on what @TallTed has already stated. You should only be concerned if they aren't working, otherwise, they are usable solid servers etc..

**Comment by ewingson** [comment]
@kidehen @csarven @TallTed @melvincarvalho @-maintainers-

Thank you. I have added them.

File: pages/use-solid.md
Comment (TallTed) [inline pages/use-solid.md:52]:
```suggestion
| [OpenLink Software (1)](https://solid.openlinksw.com:8444/) | [OpenLink Software](https://www.openlinksw.com/)                   | [Amazon](https://aws.amazon.com)              | USA              | [NSS fork](https://github.com/OpenLinkSoftware/node-solid-server)  |
| [OpenLink Software (2)](https://solid.openlinksw.com:8445/) | [OpenLink Software](https://www.openlinksw.com/)                   | [Amazon](https://aws.amazon.com)              | USA              | [NSS fork](https://github.com/OpenLinkSoftware/node-solid-server)  |
```

**Review by michielbdejong — APPROVED** [review]

**Pull request merged**
